### PR TITLE
fix(api): junit compute

### PR DESCRIPTION
**Pull request by yesnault**
Signed-off-by: Yvonnick Esnault <yvonnick.esnault@corp.ovh.com>

### CODE REVIEW

File: engine/worker/builtin_junit.go [hunk 1]
```diff
@@ -124,7 +165,7 @@ func runParseJunitTestResultAction(a *sdk.Action, pbJob sdk.PipelineBuildJob, st
 		sendLog(pbJob.ID, res.Reason, pbJob.PipelineBuildID, stepOrder, false)
```
Comment (sguiheux) [inline engine/worker/builtin_junit.go]: res.Status = Fail

File: engine/worker/builtin_junit.go [hunk 2]
```diff
@@ -85,28 +84,70 @@ func runParseJunitTestResultAction(a *sdk.Action, pbJob sdk.PipelineBuildJob, st
 	// update global stats
 	for _, ts := range v.TestSuites {
```
Comment (fsamin) [inline engine/worker/builtin_junit.go]: may this code be in a single function; and have unit tests (table unit tests) on this algorithm

**Comment by fsamin** [comment]
build_test.go:7:2: cannot find package "github.com/docker/docker/pkg/testutil/assert" in any of:
/work/src/github.com/ovh/cds/vendor/github.com/docker/docker/pkg/testutil/assert (vendor tree)
/usr/local/go/src/github.com/docker/docker/pkg/testutil/assert (from $GOROOT)
/work/src/github.com/docker/docker/pkg/testutil/assert (from $GOPATH)

File: engine/worker/builtin_junit.go (hunk 1 above)
Comment (yesnault) [inline engine/worker/builtin_junit.go]: fixed

**Pull request merged**